src/main/webapp/js/instructorFeedbackEdit/customFeedbackPaths.js

```diff
@@ -0,0 +1,358 @@
+                giverToRecipientsMap[giver].push(TEAM_NAME_INSTRUCTORS);
+            } else if (giverType === FEEDBACK_PARTICIPANT_TYPE_STUDENTS) {
+                giverToRecipientsMap[giver].push(studentEmailToTeamNameMap[giver]);
+            }
+        }
+    }
+}
+
+function populateGiverToRecipientsMapForRecipientAsOwnTeamMembers(giverToRecipientsMap, giverType) {
+    if (giverType === FEEDBACK_PARTICIPANT_TYPE_STUDENTS) {
+        for (var giver in giverToRecipientsMap) {
+            if (giverToRecipientsMap.hasOwnProperty(giver)) {
+                var giverTeamName = studentEmailToTeamNameMap[giver];
+                var giverTeamMembers = teamNameToStudentEmailsMap[giverTeamName];
+                for (var i = 0; i < giverTeamMembers.length; i++) {
+                    if (giverTeamMembers[i] !== giver) {
+                        giverToRecipientsMap[giver].push(giverTeamMembers[i]);
+                    }
+                }
+            }
+        }
+    }
+}
+
+function populateGiverToRecipientsMapForRecipientAsOwnTeamMembersIncludingSelf(giverToRecipientsMap, giverType) {
+    for (var giver in giverToRecipientsMap) {
+        if (giverToRecipientsMap.hasOwnProperty(giver)) {
+            if (giverType === FEEDBACK_PARTICIPANT_TYPE_STUDENTS) {
+                var giverTeamName = studentEmailToTeamNameMap[giver];
+                giverToRecipientsMap[giver] = teamNameToStudentEmailsMap[giverTeamName];
+            } else if (giverType === FEEDBACK_PARTICIPANT_TYPE_TEAMS) {
+                giverToRecipientsMap[giver] = teamNameToStudentEmailsMap[giver];
+            }
+        }
+    }
+}
+
+function populateGiverToRecipientsMapForRecipientAsNobodySpecific(giverToRecipientsMap) {
+    for (var giver in giverToRecipientsMap) {
+        if (giverToRecipientsMap.hasOwnProperty(giver)) {
+            giverToRecipientsMap[giver].push('%GENERAL%');
+        }
+    }
+}
+
+function bindEventHandlers() {
+    $('.form_question').on('change', '.participantSelect', function() {
+        var $questionForm = $(this).closest('.form_question');
+        var $container = $questionForm.find('.custom-feedback-paths-spreadsheet');
+        var $giverSelect = $questionForm.find('select[id^="' + FEEDBACK_QUESTION_GIVERTYPE + '"]');
+        var $recipientSelect = $questionForm.find('select[id^="' + FEEDBACK_QUESTION_RECIPIENTTYPE + '"]');
+        var giverType = $giverSelect.val();
+        var recipientType = $recipientSelect.val();
+        
+        if (giverType === FEEDBACK_PARTICIPANT_TYPE_CUSTOM
+                || recipientType === FEEDBACK_PARTICIPANT_TYPE_CUSTOM) {
+            $giverSelect.find('option[value="' + FEEDBACK_PARTICIPANT_TYPE_CUSTOM + '"]').remove();
+            $recipientSelect.find('option[value="' + FEEDBACK_PARTICIPANT_TYPE_CUSTOM + '"]').remove();
+        }
+        
+        giverType = $giverSelect.val();
+        recipientType = $recipientSelect.val();
```

Comment:
Yup it uses the default value, generally this would be the first non-disabled option. If I use a standard default, I would need to consider the case where the question is a Team Contrib question as it only allows students to giver's team members and giver as the predefined feedback path. This might make the logic more complicated than necessary.